src/pages/tug/apply/index.page.js

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
+
+import Apply from './Apply.component';
+import Error from '../../_error.page';
```

Comment:
If the relative path is deeper than two layers, I'd suggest using the absolute path as

```javascript
import ErrorPage from '~/pages/_error.page`;
```

Please check the definitions of alias in https://github.com/pingcap/tug-website/blob/main/next.config.js#L39-L40